addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10976,7 +10976,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#20369"
-msgid "Show plot for unwatched items"
+msgid "Show plot and episode thumbnail for unwatched items"
```

Comment:
This should be separate options because of the need to see the plot for movies but not see spoiler revealing thumbs for episodes. Just my 2 cents.